### PR TITLE
Fix conjugates order for panels with multiple targets on the same mass

### PR DESCRIPTION
**Pull request by stanislaw-ionpath**

### CODE REVIEW

File: mibidata/panels.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 from mibidata import util
 
 
+def pascal_case(st):
```
Comment (murataksoy) [inline mibidata/panels.py]: Doc?

File: mibidata/panels.py [hunk 2]
```diff
@@ -8,6 +8,10 @@
...
+    return ''.join(x for x in st.title() if x.isalnum())
+
```
Comment (murataksoy) [inline mibidata/panels.py]: Is this OK?:
```
In [5]: aaa="β-tubulin"                                                                                                                   

In [6]: aaa                                                                                                                               
Out[6]: 'β-tubulin'

In [7]: aaa.title()                                                                                                                       
Out[7]: 'Β-Tubulin'

```

Comment (murataksoy) [reply mibidata/panels.py]: Ah, I see now what you are doing - this is just about "target" and "mass". OK

File: mibidata/panels.py (hunk 1 above)
Comment (rfinck) [reply mibidata/panels.py]: Should this be a private function?

**Pull request closed without merging**